packages/form-js-playground/test/spec/Playground.spec.js

```diff
@@ -790,6 +791,60 @@ describe('playground', function() {
 
   });
 
+
+  describe('a11y', function() {
+
+    it('should have no violations', async function() {
+
+      // given
+      this.timeout(5000);
```

Comment:
Why is the timeout necessary?

Reply:
`axe` takes a while to run its validations, leading `karma` to run out of time sometimes (flaky).